modules/search.py

```diff
@@ -0,0 +1,73 @@
+from elasticsearch import Elasticsearch, ElasticsearchException
+
+class SearchError(Exception):
+    pass
```

Comment:
🤔

Reply:
For the time being we just need the existence of the error, not any additional information.